Command/LoadDataFixturesDoctrineCommand.php

```diff
@@ -75,13 +79,25 @@ protected function execute(InputInterface $input, OutputInterface $output)
             }
         }
 
+        $fixturesPath = DIRECTORY_SEPARATOR . 'DataFixtures' . DIRECTORY_SEPARATOR . 'ORM';
         $dirOrFile = $input->getOption('fixtures');
         if ($dirOrFile) {
-            $paths = is_array($dirOrFile) ? $dirOrFile : array($dirOrFile);
+
+            $paths = array();
+            $pathsParams = is_array($dirOrFile) ? $dirOrFile : array($dirOrFile);
+            foreach ($pathsParams as $path) {
+                if (preg_match('/^\@([^\/\\\\]+)(.*)$/', $path, $matches)) {
+                    $bundle = $this->getApplication()->getKernel()->getBundle($matches[1]);
```

Comment:
When the bundle does not exist (or typo), is it handled properly? There should be a test case for that.